recipe/meta.yaml

```diff
@@ -154,16 +153,48 @@ test:
     - pyomo.solvers.plugins.solvers
     - pyomo.util
     - pyomo.version
-
+  source_files:
+  # Tests are spread across many files throughout the source code.
+    - .
+  requires:
+    - pip
+    - pytest
+    - parameterized
+    - pybind11
```

Comment:
Why is pybind11 needed to run the tests?

Reply:
Unsure, I was just following the upstream testing dependencies here: https://github.com/Pyomo/pyomo/blob/1ada528693d8879f17978fe47263f8bb2b467b0a/setup.py#L249

I can give it a shot without pybind11.